docs/articles/documentation/reference/test-api/testcontroller/setpageloadtimeout.md

```diff
@@ -32,4 +34,4 @@ test(`Page load timeout`, async t => {
 });
 ```
 
-> Note that browsers raise the `DOMContentLoaded` event after they load and parse the HTML document, while `window.load` is raised after the browser loads all stylesheets, images and subframes. This is why `window.load` fires after the `DOMContentLoaded` event with a certain delay.
+> Browsers raise the `DOMContentLoaded` event after they load and parse the HTML document, while `window.load` is raised after the browser loads all stylesheets, images and subframes. This is why `window.load` fires after the `DOMContentLoaded` event with a certain delay.
```

Comment:
a) This side note might seem irrelevant at first glance, so let's get straight to the point. 
b) I'd avoid using "the HTML document" to describe the HTML content of a page - it's not always true these days

```suggestion
> The `DOMContentLoaded` event always precedes the `window.load` event. It fires as soon as the browser loads and parses the HTML content of the page. The `window.load` event, on the other hand, waits for the browser to load all the style sheets, images and sub-frames.
```

Reply:
Can an **event** really wait for anything?

```suggestion
> The `DOMContentLoaded` event always precedes the `window.load` event. It fires when the browser has loaded and parsed the HTML content of the page. The `window.load` event, on the other hand, fires when the browser has loaded all the style sheets, images and sub-frames.
```